src/lib/mergeThemes.ts

```diff
@@ -64,9 +65,10 @@ export const mergeComponentVariables = (
     return (...args) => {
       const accumulatedVariables = acc(...args)
       const computedComponentVariables = callable(next)(...args)
+      const mergedVariables: ObjectOf<any> = {}
 
-      const mergedVariables = {}
-      _.mapKeys(computedComponentVariables, (variableToMerge, variableName) => {
+      Object.keys(computedComponentVariables).forEach(variableName => {
+        const variableToMerge = computedComponentVariables[variableName]
```

Comment:
This assumes `computedComponentVariables` exists and is an object.  If it is undefined or null, this will also throw. 
 Suggest sticking with more lodash here to safe guard ourselves.

Note, `_.forEach` will iterate `(val, key) => {}` just like `_.mapKeys` but doesn't have a return value so it is arguably more appropriate in this case.  Essentially doing the same as `Object.keys().forEach()` except in a safe manner.

Reply:
done, thanks